### ISSUE TITLE
Repo size

### ISSUE THREAD

**Issue by tpb1908** [tpb1908/FaceDetector#17]
The repo is now up to ~260MB and increasing, and will only get bigger.

I think the simplest way to store the files is as a submodule.

When FaceDetector is pulled, git should check if the Files repo needs to be updated and only pull it when it has been updated.

**Comment by zoeyfyi** [comment]
Absolutely but we need to make sure your script works so we can't be to
fancy!

**Comment by tpb1908** [comment]
Yeah, this is obviously to be done once everything else is fixed.

If a submodule doesn't work, then we can just host the files somewhere and throw an error (Or even get Python to download them) if they don't exist.
